### ISSUE TITLE
Inconsistent white space for template XCode SwiftUI project straight out of the box

### ISSUE THREAD

**Issue by gigecogary** [nicklockwood/SwiftFormat#767]
The following gives the error inconsistent white space, at line 19
This is a file that is generated by xcode for a brand new SwiftUI project

```
//
//  ContentView.swift
//  Test SwiftUI
//
//

import SwiftUI

struct ContentView: View {
    var body: some View {
        Text("Hello, world!")
            .padding()
    }
}

struct ContentView_Previews: PreviewProvider {
    static var previews: some View {
        ContentView()
    }
}

```

**Comment by bencochran** [comment]
It looks like there’s a form feed character (U+000C) after `ContentView()`. Are you sure this wasn’t modified by mistake after being generated? I just made a new SwiftUI app with Xcode 12.0 (12A7209) and got no unexpected character in that same location.

**Comment by gigecogary** [comment]
You are right, closing this issue.

**Comment by nicklockwood** [comment]
If U+000C is permitted by the Swift compiler, it probably shouldn't cause an error in SwiftFormat. I'll look at fixing that.

**Comment by nicklockwood** [comment]
@gigecogary fixed in 0.47.0
